audit-service/README.md

```diff
@@ -38,11 +45,21 @@ been implemented. This service does not have any output Kafka topics.
 
 *The token value come from the headers of the Kafka message that the service receives. This links the audit message to the original request that was made.
 
-If an error has occurred during a request/read then an AuditErrorMessage will be consumed from the `error` Kafka topic. This type of message can be sent from any of the Palisade services
-(e.g user-service or policy-service).
+If an error has occurred at any stage during either the request or read phases then an AuditErrorMessage will be added to the `error` Kafka topic by the service that encountered the issue. 
+This type of message can be sent from any of the Palisade services (e.g. User Service or Policy Service).
```

Comment:
Needs to be clarified.  The text of "during either the request or read phases" could be better described.  It suggests that the error messages are limited in the context of what they report.  The error messages can occur in any step of processing the request throughout Palisade Services.   
The word any in "can be sent from any of the Palisade Services" should be replaced with all.   Suggest either listing all of the services or none as listing two is misleading.
Also, e.g. User Service or Policy Service

Reply:
I've updated this part so that it reads differently.